src/users/users.controller.ts

```diff
@@ -0,0 +1,33 @@
+import {
+    Controller,
+    Get,
+    Param,
+    ParseBoolPipe,
+    ParseIntPipe,
+    Query,
+    UsePipes,
+    ValidationPipe,
+} from '@nestjs/common'
+import { Pageable } from 'src/interfaces/pageable.interface'
+import { QueryUserDto } from './dto/get-user.dto'
+import { UserEntity } from './entities/user.entity'
+import { UsersService } from './users.service'
+
+@Controller('users')
+export class UsersController {
+    constructor(private readonly usersService: UsersService) {}
+
+    @Get()
+    @UsePipes(new ValidationPipe({ transform: true }))
```

Comment:
remove this

Reply:
`@UsePipes(new ValidationPipe({ transform: true}))` is working with `@Transform(({ value})=> parseInt(value))` to change type of value when sending through query params. If you prefer to remove this code, I think we need to find a solution for working with another data type except string.